src/domain/wallet-invoices/index.types.d.ts

```diff
@@ -84,8 +89,15 @@ type WalletInvoiceReceiver = WalletInvoice & {
   receivedAmount: () => BtcPaymentAmount | UsdPaymentAmount
 }
 
-type WalletInvoiceReceiverArgs = {
-  walletInvoice: WalletInvoice
+type WalletAddressReceiver<S extends WalletCurrency> = WalletAddress<S> & {
```

Comment:
Something seems wrong with this type. Don't quite understand what yet.
Not sure of the name or the bundling of the arguments.

Reply:
I looked at the original `WalletInvoiceReceiver` type again and made some changes. Do these help? https://github.com/GaloyMoney/galoy/compare/2c87ed30f2d0be6b86b3057683d467fde0ed77f9...60fb5fa5d03bf55d513c5bf69d5e72d6405e47b6